### PR TITLE
[proto] Clean-ups from 6512

### PR DESCRIPTION
**Pull request by vfdev-5**
- Cherrypicked 2 commits from #6512 with clean-ups to merge quicker
- Tests fix + renamed `_compute_output_size` methods with a better names

### CODE REVIEW

File: torchvision/transforms/functional_tensor.py [hunk 1]
```diff
@@ -610,7 +610,7 @@ def affine(
     return _apply_grid_transform(img, grid, interpolation, fill=fill)
 
 
-def _compute_output_size(matrix: List[float], w: int, h: int) -> Tuple[int, int]:
+def _compute_affine_output_size(matrix: List[float], w: int, h: int) -> Tuple[int, int]:
```
Comment (vfdev-5) [inline torchvision/transforms/functional_tensor.py:613]: @datumbox please check this renaming to solve colliding function names and comment out if you approve or disagree. Thanks!

File: torchvision/transforms/functional.py [hunk 2]
```diff
@@ -360,7 +360,9 @@ def normalize(tensor: Tensor, mean: List[float], std: List[float], inplace: bool
     return F_t.normalize(tensor, mean=mean, std=std, inplace=inplace)
 
 
-def _compute_output_size(image_size: Tuple[int, int], size: List[int], max_size: Optional[int] = None) -> List[int]:
+def _compute_resized_output_size(
```
Comment (vfdev-5) [inline torchvision/transforms/functional.py:363]: @datumbox please check this renaming to solve colliding function names and comment out if you approve or disagree. Thanks!

**Review by pmeier — APPROVED** [review]
LGTM if CI is green.

File: torchvision/transforms/functional_tensor.py (hunk 1 above)
Comment (datumbox) [reply torchvision/transforms/functional_tensor.py:613]: Sounds good, thx!

**Pull request merged**